### PR TITLE
Object detection example rework

### PR DESCRIPTION
**Pull request by aga11313**

### CODE REVIEW

**Review by FionnD — CHANGES_REQUESTED** [review]
This is still in progress. I can't get the code running but I'll open an PR to see if someone else can. I can come back and review it once that issue is fixde.

File: object-search/README.md [hunk 1]
```diff
@@ -1,182 +1,186 @@
...
 
-If it's running successfully, you should be able to see logs scrolling in the console and in the dashboard:
+To index the entire dataset simply run: (This could take a lot longer)
+```
+python app.py -task index -n 8000 -overwrite True
+```
+
+The Indexing process for this example is as follows:
+1. Go through all of the images and identify various objects contained.
+2. Identify the cropped sub images and their labels.
```
[the remark this answers is not in the document]
Comment (aga11313) [reply object-search/README.md]: Okay, let me think about this some more

**Comment by FionnD** [comment]
Reviewing this is blocked by https://github.com/jina-ai/examples/issues/519

**Comment by cristianmtr** [comment]
@FionnD the example works for me now.

I cannot merge with `master` since there are too many changes in the README, and I'm not up to speed with what is the current vision. Should I just replace the changes in `master` with the ones in this PR?

**Comment by FionnD** [comment]
Thanks @cristianmtr 

I don't quite understand what you mean, should you just move the app.py changes to another PR is it?

**Comment by cristianmtr** [comment]
The PR is showing conflicting changes that cannot be automatically resolved in merging:

image

I tried to rebase. The app.py merges fine, but the README has too many changes. And I'm not sure which is the way to go

**Comment by cristianmtr** [comment]
I reverted my push to this PR. My bad, I thought this PR was for reworking the code itself too

**Comment by nan-wang** [comment]
resolved the conflicts

**Pull request merged**